internal/contour/cluster.go

```diff
@@ -15,66 +15,28 @@ package contour
 
 import (
 	"sort"
-	"sync"
-
-	resource "github.com/envoyproxy/go-control-plane/pkg/resource/v2"
 
 	v2 "github.com/envoyproxy/go-control-plane/envoy/api/v2"
-	"github.com/golang/protobuf/proto"
+	xds "github.com/envoyproxy/go-control-plane/pkg/cache/types"
 	"github.com/projectcontour/contour/internal/dag"
 	"github.com/projectcontour/contour/internal/envoy"
 	"github.com/projectcontour/contour/internal/protobuf"
 	"github.com/projectcontour/contour/internal/sorter"
 )
 
-// ClusterCache manages the contents of the gRPC CDS cache.
-type ClusterCache struct {
-	mu     sync.Mutex
-	values map[string]*v2.Cluster
-	Cond
-}
-
-// Update replaces the contents of the cache with the supplied map.
-func (c *ClusterCache) Update(v map[string]*v2.Cluster) {
-	c.mu.Lock()
-	defer c.mu.Unlock()
+// ClusterTranslator manages the contents of the gRPC CDS translator.
+type ClusterTranslator struct{}
```

Comment:
any particular reason to keep this as a struct? Seems like it could just be a package function since there's no state in the struct anymore.